modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/user/store/v1/model/UserStoreReq.java

```diff
@@ -0,0 +1,347 @@
+                    Objects.equals(this.value, property.value);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(name, value);
+        }
+
+        @Override
+        public String toString() {
+
+            return "class Property {\n" +
+                    "    name: " + toIndentedString(name) + "\n" +
+                    "    value: " + toIndentedString(value) + "\n" +
+                    "}";
+        }
+    }
+
+    public static class ClaimAttributeMapping  {
+
+        private String claimURI;
+        private String mappedAttribute;
+
+        /**
+         * A unique URI specific to the claim.
+         **/
+        public ClaimAttributeMapping claimURI(String claimURI) {
+
+            this.claimURI = claimURI;
+            return this;
+        }
+
+        @ApiModelProperty(example = "http://wso2.org/claims/username", required = true, value = "A unique URI specific to the claim.")
+        @JsonProperty("claimURI")
+        @Valid
+        @NotNull(message = "Property claimURI cannot be null.")
+        public String getClaimURI() {
+            return claimURI;
+        }
+        public void setClaimURI(String claimURI) {
+            this.claimURI = claimURI;
+        }
+
+        /**
+         * Userstore attribute to be mapped to.
+         **/
+        public ClaimAttributeMapping mappedAttribute(String mappedAttribute) {
+
+            this.mappedAttribute = mappedAttribute;
+            return this;
+        }
+
+        @ApiModelProperty(example = "username", required = true, value = "Userstore attribute to be mapped to.")
+        @JsonProperty("mappedAttribute")
+        @Valid
+        @NotNull(message = "Property mappedAttribute cannot be null.")
+        public String getMappedAttribute() {
+            return mappedAttribute;
+        }
+        public void setMappedAttribute(String mappedAttribute) {
+            this.mappedAttribute = mappedAttribute;
+        }
+
+        @Override
+        public boolean equals(java.lang.Object o) {
+
+            if (this == o) {
+                return true;
+            }
+            if (o == null || getClass() != o.getClass()) {
+                return false;
+            }
+            ClaimAttributeMapping claimAttributeMapping = (ClaimAttributeMapping) o;
+            return Objects.equals(this.claimURI, claimAttributeMapping.claimURI) &&
+                    Objects.equals(this.mappedAttribute, claimAttributeMapping.mappedAttribute);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(claimURI, mappedAttribute);
+        }
+
+        @Override
+        public String toString() {
+
+            return "class ClaimAttributeMapping {\n" +
+                    "    claimURI: " + toIndentedString(claimURI) + "\n" +
+                    "    mappedAttribute: " + toIndentedString(mappedAttribute) + "\n" +
+                    "}";
+        }
+    }
+}
```

Comment:
Add new line.

Reply:
Fixed with https://github.com/wso2/product-is/pull/16354/commits/8796c767b55bfb43464a54ea8f1ddb369a1a1401